mautic/mautic
# Segment detail view

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR add a detail view for segments, with a stats chart of contacts added in time and a contacts tab with filters for manually added contacts, filter added contacts and manually removed contacts
[//]: # ( As applicable: )

#### Steps to test this PR:
1. Go to the segment list and click on a segment you should now see a detail page
2. Test the filters for the graph correct number or contacts should appear
3. Test the filters for the contacts tab, you should be able to see who was manually added, manually removed and filter added. no filters lists all active contacts.
